tools/topology/topology2/include/components/src_format_s32_convert_from_48k.conf

```diff
@@ -1,95 +1,28 @@
 #src format array
 			num_output_audio_formats 14
 
-			Object.Base.output_audio_format [
-				# 8khz output
-				{
-					out_rate                 8000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 11.025 khz output
-				{
-					out_rate                 11025
-					out_bit_depth            32
-					out_valid_bit_depth      32
-					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
```

Comment:
if SRC is run as DP it will be scheduled when 
1) has at least IBS bytes of data on input 
2) has at least OBS free bytes of data on output

so IBS should be exact 
OBS is less important - as buffer is allocated of size 2*MAX(ibs_of_next_module, OBS_of_current_module)
so + 999 is a waste of memory - please set precise value


i.e. if SRC is changing 16000/16/2 to 48000/16/2 and you want to process data in 1ms periods
   IBS = 64, OBS = 192

in the same processing, but if you want to process in 10ms chunks:
IBS=640 OBS=1920

Reply:
Thanks, @marcinszkudlinski , @singalsu , what's your comments on this? I would like use a separate patch to update obs, in case there is regression for new setting.

Reply:
@btian1 I asgree, this PR is purely for the purpose of using combinearray definition instead of separate audio formats. So its important to keep it identical to the existing audio formats. Any changes to the ibs/obs should be a follow up

Reply:
and it is retained with this PR @kv2019i

Reply:
I stand corrected, so it is.